src/launcher/actions/userDataActions.js

```diff
@@ -85,6 +86,13 @@ export function setUserDataOff() {
     };
 }
 
+export function resetUserData() {
+    settings.set('isSendingUserData', undefined);
```

Comment:
I'd recommend to set it to false rather than undefined.

Reply:
I intentionally do this. Setting to true or false will make the dialog always hide. Setting to other values will make it display at launch time.